components/mpas-ocean/cime_config/config_pes.xml

```diff
@@ -506,7 +506,7 @@
     </mach>
   </grid>
   <grid name="a%T62.+oi%oARRM60to10|a%TL319.+oi%oARRM60to10|a%T62.+oi%ARRM10to60|a%TL319.+oi%ARRM10to60">
```

Comment:
@apcraig I think it's not picking up any default PE layout for the G-cases because this is the old ocn mesh name, if we replace `ARRM60to10` with `ARRM10to60E2r1` here I think it should pick it up.

Reply:
But the DoD machines are picking up the pe layouts fine.  The difference is that there is a "pm-cpu" default at line 75 of components/mpas-ocean/cime_config/config_pes.xml that is somehow taking precedent.  Another case of the scripts not being entirely transparent about how they work or what they're doing.  My sense is a handful of people know how they work and the rest of us are just hoping.